integration/command.go

```diff
@@ -123,6 +123,16 @@ for i in $(seq 1 120); do
   sleep 1
 done
 
+# Similar to https://github.com/Azure/AKS/issues/1771 SPO webhook reconcilation doesn't work
+# on AKS because of an additional selector. So we patch the SPO-daemon config to respect this selector
+# using https://github.com/kubernetes-sigs/security-profiles-operator/blob/main/installation-usage.md#configuring-webhooks.
+# SPO issue: https://github.com/kubernetes-sigs/security-profiles-operator/issues/1187
+if [ %v = true ] ; then
```

Comment:
hmm, I thought I ready answered to that already. In the context of entire script I find doing this `bash` bit more readable since we are doing it in 3 different places. WDYT?